### PR TITLE
[Merged by Bors] - Fix SmartStream label, update CHANGELOG

### PR DESCRIPTION
**Pull request by nicholastmosher**

### CODE REVIEW

File: src/smartstream/derive/src/generator/map.rs [hunk 1]
```diff
@@ -49,7 +49,7 @@ pub fn generate_map_smartstream(func: &SmartStreamFn) -> TokenStream {
                             let error = fluvio_smartstream::dataplane::smartstream::SmartStreamRuntimeError::new(
                                 &record,
                                 smartstream_input.base_offset,
-                                fluvio_smartstream::dataplane::smartstream::SmartStreamType::Filter,
+                                fluvio_smartstream::dataplane::smartstream::SmartStreamType::Map,
```
Comment (sehz) [inline src/smartstream/derive/src/generator/map.rs:52]: there should be test to catch this issue.

**Review by sehz — CHANGES_REQUESTED** [review]

File: src/smartstream/derive/src/generator/map.rs (hunk 1 above)
Comment (nicholastmosher) [reply src/smartstream/derive/src/generator/map.rs:52]: I opened an issue, can we handle later? https://github.com/infinyon/fluvio/issues/1356

Comment (sehz) [reply src/smartstream/derive/src/generator/map.rs:52]: this kind of major issue can't be fell thru crack.  let's complete the test

Comment (nicholastmosher) [reply src/smartstream/derive/src/generator/map.rs:52]: Added test

**Review by sehz — APPROVED** [review]
LGTM

**Pull request closed without merging**